py/torch_tensorrt/dynamo/runtime/_WrapperTorchTensorRTModule.py

```diff
@@ -0,0 +1,271 @@
+from __future__ import annotations
+
+import logging
+from contextlib import nullcontext
+from tempfile import tempdir
+from typing import List, Optional, Sequence, Tuple
+
+import torch
+import torch_tensorrt
+from torch.fx.experimental.proxy_tensor import unset_fake_temporarily
+from torch_tensorrt.dynamo import partitioning
+from torch_tensorrt.dynamo.conversion import DYNAMIC_DIM
+from torch_tensorrt.dynamo.utils import input_is_dynamic
+from torch_tensorrt.runtime._utils import _is_switch_required, _select_rt_device
+
+logger = logging.getLogger(__name__)
+
+
+class WrapperTorchTensorRTModule(torch.nn.Module):  # type: ignore[misc]
+    """This Wrapper runtime module to record/replay cuda graph in sub modules"""
+
+    def __init__(
+        self,
+        original_module: torch.nn.Module,
+        output_shapes: List[torch.Size],
+        output_dtypes: List[torch.dtype],
+    ):
+        super(WrapperTorchTensorRTModule, self).__init__()
+        self.original_module = original_module
```

Comment:
Would this module be the original PyTorch one or the compiled one?

Reply:
It is compiled module. I renamed it to be descriptive.